### ISSUE TITLE
First pull is not working

### ISSUE THREAD

**Issue by jllodra** [normanhh3/jqm-iscroll#1]
Hi, I've tried the example you provide, and the first time I pull it seems not to work.
But if I repeat the pull movement, it does work. Is this happening to you too?

**Comment by normanhh3** [comment]
Were you able to run the Meteor demo by any chance? Is it possible that there is a timing issue involved?

**Comment by jllodra** [comment]
Yes, I ran the Meteor project included. It could be a timing issue, but I tried to $('#scroller').iscroll('refresh'); with a setTimeout and did not fix the problem, the first time I pull down, it doesn't work.

**Comment by normanhh3** [comment]
Let me poke at it and see if I can replicate it and get back to you.

**Comment by normanhh3** [comment]
Good news.. I was able to replicate the issue you are having.  Let me see
if there was something that I did with my current production code that
might explain why this is behaving how it is.  I'll need to get back to you.
